django_perf_rec/api.py

```diff
@@ -15,15 +17,30 @@
 record_current = local()
 
 
-def record(file_name=None, record_name=None):
+def record(file_name=None, record_name=None, path=None):
     test_details = current_test()
 
-    if file_name is None:
-        file_name = test_details.file_path
-        if file_name.endswith('.py'):
-            file_name = file_name[:-len('.py')] + '.perf.yml'
+    if file_name is not None:
+        warnings.warn(
+            '"file_name" is deprecated, use the "path" argument instead',
+            DeprecationWarning
+        )
+    else:
+        if path is None or path.endswith('/'):
+            file_name = test_details.file_path
+            if file_name.endswith('.py'):
+                file_name = file_name[:-len('.py')] + '.perf.yml'
+            else:
+                file_name += '.perf.yml'
         else:
-            file_name += '.perf.yml'
+            file_name = path
+
+        if path is not None and path.endswith('/'):
+            directory = os.path.join(os.path.dirname(test_details.file_path), path)
+            if not os.path.exists(directory):
+                os.mkdir(directory)
```

Comment:
This doesn't deal with nested dirs, I've added that capability by using `makedirs` and adding a corresponding test